index.src.html

```diff
@@ -577,10 +579,28 @@ <h3 id="clear-response">
       <a href="https://lists.w3.org/Archives/Public/public-webappsec/2015Jun/0032.html">Martin
       Thomson's public-webappsec post on the topic</a>, for example).
 
-  2.  Let |types| be the result of [[#get-types]] executed on
+  2.  If |response| was returned by a service worker <a>FetchEvent</a>, skip
```

Comment:
If we want this to be a network-only thing (and I think we do in order to prevent a service worker from impersonating other origins, or caching their responses for later use), then a better option is to move the hook to after step 14 of https://fetch.spec.whatwg.org/#http-network-fetch (which deals with `Set-Cookies`). That only triggers when we're actually talking to the network, and seems like a better way of addressing the issue than by referring to global state that we're not actually storing on the response.

Again, @annevk might have a better integration suggestion, but moving the hook makes sense to me.